cmake/toolchain/clang.riscv.cmake

```diff
@@ -9,5 +9,5 @@ set(CMAKE_SYSTEM_PROCESSOR  riscv64)
 set(CMAKE_C_COMPILER    /opt/llvm/bin/clang)
 set(CMAKE_CXX_COMPILER  /opt/llvm/bin/clang++)
 
-set(CMAKE_CXX_FLAGS "-march=rv64gcv --static --target=riscv64-unknown-linux-gnu ${EVE_OPTIONS}" )
+set(CMAKE_CXX_FLAGS "-march=rv64gcv --static -fbracket-depth=512 --target=riscv64-unknown-linux-gnu ${EVE_OPTIONS}" )
```

Comment:
what does it control ?

Reply:
This is the depth limit for nested parentheses, brackets, and braces. By default it is set to 256, but as we have a lot of inline, we reach this limit for simd_cast test.